packages/cpp/README.md

```diff
@@ -102,6 +102,43 @@ There are two ways to configure clang-tidy's checks: through a Theia preference
     - for the .clang-tidy file: Checks: "-*,readability-*"
         - Meaning: disable all list-checks and enable all readability-* checks
 
+### Running clang-tidy as a task
+
+To be able to run clang-tidy as a task, you need to install the program.
+
+```bash
+UNIX
+    sudo apt-get install clang-tidy
+Note: not all operating system are currently supported with this linter yet.
+```
+The clang-tidy task will read the checks from the ".clang-tidy" file located in the root folder.
```

Comment:
For the sake of completeness, the `.clang-tidy` file does not only describe checks, and `clang-tidy` as a task does not only read the checks from the file. There are other configuration options that can be provided and read (ex: `WarningsAsChecks`), so I don't believe that we should only state that it will read the checks.

Sorry if its nitpicking, i just want the documentation to be complete and easy to follow. Throughout the PR review I had a lot of issues initially just reading the documentation and getting it working.

Reply:
@vince-fugnitto @elaihau  Readme file adjusted to show more information about what can be found in .clang-tidy file